iib/web/models.py

```diff
@@ -1092,6 +1092,8 @@ def from_json(cls, kwargs, batch=None):
         if not isinstance(from_bundle_image, str):
             raise ValidationError('"from_bundle_image" must be a string')
 
+        # Remove attributes that are not stored in the database
+        request_kwargs.pop('registry_auths', None)
```

Comment:
We should validate that if `registry_auths` is used, it conforms to the expected format.

Reply:
I added check if it's a dict, not sure if we need a more sophisticated check here.

Reply:
Do you want check if there is 'auths' on top level and then registries?

Reply:
The `.dockerconfig.json` format allows for things other than auth information to be set. We should prevent those from being used. I think the most straightforward way is to raise a validation error if those are set. So, I propose you check the following:

1. `registry_auths` is of type dict
2. `regsitry_auths` contain a single key, `auths`
3. `regsitry_auths.auths` is a dict, it could be empty IMO
4. each _value_ in `auths` is a dict and must have one and only key named `auth`. The value of `auth` should be a string.

This will look verbose, but we don't really have another option. At some point we should revisit using a json schema to do this sort of thing.

Reply:
I added validation check. Looks terrible and verbose. Please take a look if it looks usable @lcarva . I suppose unit test is required. :)

Reply:
It doesn't look that bad :)